### ISSUE TITLE
Copy/Paste Regressions

### ISSUE THREAD

**Issue by dhirschfeld** [jupyter/qtconsole#48]
I've noticed a couple of regressions with the latest qtconsole:
1. The last character of a selection isn't copied - see also #47
2. The prompts aren't stripped on copy
![qtconsole](https://cloud.githubusercontent.com/assets/881019/9508397/569c6068-4c98-11e5-8f7a-fcd6f9d2adf0.gif)
```python
In [10]: print sys_info()
-> print(sys_info())
{'commit_hash': 'f534027',
 'commit_source': 'installation',
 'default_encoding': 'cp1252',
 'ipython_path': 'C:\\Anaconda3\\lib\\site-packages\\IPython',
 'ipython_version': '4.0.0',
 'os_name': 'nt',
 'platform': 'Windows-8-6.2.9200',
 'sys_executable': 'C:\\Anaconda3\\python.exe',
 'sys_platform': 'win32',
 'sys_version': '3.4.3 |Anaconda 2.3.0 (64-bit)| (default, Mar  6 2015, '
                '12:06:10) [MSC v.1600 64 bit (AMD64)]'}

In [11]: !conda list "jupyter|ipython"
# packages in environment at C:\Anaconda3:
#
ipython                   4.0.0                    py34_0  
ipython-genutils          0.1.0                     <pip>
ipython_genutils          0.1.0                    py34_0  
jupyter                   1.0.0                    py34_0  
jupyter-client            4.0.0                     <pip>
jupyter-console           4.0.0                     <pip>
jupyter-core              4.0.4                     <pip>
jupyter_client            4.0.0                    py34_0  
jupyter_console           4.0.0                    py34_0  
jupyter_core              4.0.4                    py34_0  
```

NB: Maybe `sys_info` should be updated to show info about the whole jupyter install?

**Comment by zhangxaochen** [comment]
excellent GIF~ how is that captured? 
![1 p2 eon a_mw w654 h](https://cloud.githubusercontent.com/assets/3416677/9567319/da61a922-4f58-11e5-8d06-8a9b3babd751.gif)

**Comment by dhirschfeld** [comment]
@zhangxaochen - I use [LICEcap](http://www.cockos.com/licecap/). Can be easier than trying to describe the problem :)

**Comment by zhangxaochen** [comment]
@dhirschfeld thx~ it's cool!

**Comment by 691175002** [comment]
This is driving me absolutely insane.

**Comment by simonzack** [comment]
Same! A fix would really be appreciated.

**Comment by bergtholdt** [comment]
This is a fix against current master. I don't have the time right now to make a pull request. If someone would be so kind.

https://gist.github.com/bergtholdt/9ca387772d13da0b2c1a

**Comment by minrk** [comment]
@bergtholdt thanks! Opened as #59

**Comment by minrk** [comment]
Just released 4.1 with this fix. Thanks, everyone!
